WeakAurasTemplates/TriggerTemplatesData.lua

```diff
@@ -858,63 +872,81 @@ else
-          { spell = 132169, type = "debuff", unit = "target", talent = 6}, -- Storm Bolt
+          { spell = 355, type = "debuff", unit = "target"}, -- Taunt
+          { spell = 1715, type = "debuff", unit = "target"}, -- Hamstring
+          { spell = 12323, type = "debuff", unit = "target"}, -- Piercing Howl
+          { spell = 105771, type = "debuff", unit = "target"}, -- Charge
           { spell = 118000, type = "debuff", unit = "target", talent = 17}, -- Dragon Roar
+          { spell = 132169, type = "debuff", unit = "target", talent = 6}, -- Storm Bolt
           { spell = 280773, type = "debuff", unit = "target", talent = 21}, -- Siegebreaker
-          { spell = 105771, type = "debuff", unit = "target"}, -- Charge
-          { spell = 355, type = "debuff", unit = "target"}, -- Taunt
         },
         icon = 132154
       },
       [3] = {
         title = L["Abilities"],
         args = {
-          { spell = 100, type = "ability", requiresTarget = true, talent = {5,6}}, -- Charge    !!TODO: add prefix or name or something when 2 times same talent
-          { spell = 100, type = "ability", charges = true, requiresTarget = true, talent = 4}, -- Charge
+          { spell = 100, type = "ability", requiresTarget = true, talent = {5,6}, titleSuffix=" (1 Charge)" }, -- Charge
+          { spell = 100, type = "ability", requiresTarget = true}, -- Charge
+          { spell = 100, type = "ability", charges = true, requiresTarget = true, talent = 4, titleSuffix=" (2 Charges)"}, -- Charge
           { spell = 355, type = "ability", debuff = true, requiresTarget = true}, -- Taunt
+          { spell = 1161, type = "ability"}, -- Challenging Shout
+          { spell = 1464, type = "ability", requiresTarget = true}, -- Slam
+          { spell = 1680, type = "ability"}, -- Whirlwind
+          { spell = 1715, type = "ability", debuff = true,  requiresTarget = true}, -- Hamstring
           { spell = 1719, type = "ability", buff = true}, -- Recklessness
-          { spell = 5246, type = "ability"}, -- Intimidating Shout
+          { spell = 2565, type = "ability", buff = true, charges = true}, -- Shield Block
+          { spell = 3411, type = "ability"}, -- Intervene
+          { spell = 5246, type = "ability", debuff = true, requiresTarget = true}, -- Intimidating Shout
           { spell = 5308, type = "ability", requiresTarget = true, overlayGlow = true}, -- Execute
           { spell = 6544, type = "ability"}, -- Heroic Leap
           { spell = 6552, type = "ability", requiresTarget = true}, -- Pummel
           { spell = 6673, type = "ability"}, -- Battle Shout
-          { spell = 12323, type = "ability"}, -- Piercing Howl
+          { spell = 12323, type = "ability", debuff = true}, -- Piercing Howl
           { spell = 18499, type = "ability", buff = true}, -- Berserker Rage
+          { spell = 22398, type = "ability", talent = 17}, -- Dragon Roar
+          { spell = 22400, type = "ability", talent = 18}, -- Bladestorm
```

Comment:
![image](https://user-images.githubusercontent.com/189656/95002056-455d2200-05d0-11eb-84e4-168f4196fab9.png)
```suggestion
          { spell = 46924, type = "ability", talent = 18}, -- Bladestorm
```